### ISSUE TITLE
 Do you have a pre-packaged Docker image environment?

### ISSUE THREAD

**Issue by taoistwar** [zju3dv/EasyVolcap#12]
The installation of the environment is too cumbersome. Can you provide a pre-packaged one?

**Comment by dendenxu** [comment]
Indeed. Very good suggestion! I'll work on a docker image and try building a pypi package.
